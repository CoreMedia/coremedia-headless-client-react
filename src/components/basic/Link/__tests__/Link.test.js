// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import { Link } from '..';

describe('Link Component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Link url="https://labs.coremedia.com">Content</Link>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
