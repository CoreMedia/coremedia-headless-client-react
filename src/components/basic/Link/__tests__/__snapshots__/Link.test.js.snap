// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link Component should render correctly 1`] = `
<a
  href="https://labs.coremedia.com"
>
  Content
</a>
`;
